public/v4/apps/redmine-mysql.yml

```diff
@@ -0,0 +1,70 @@
+captainVersion: 4
+services:
+    $$cap_appname-db:
+        image: mysql:$$cap_mysql_version
+        command: --default-authentication-plugin=mysql_native_password
```

Comment:
This is being ignored by CapRover. But if you tested it, I assume this line isn't required.

Reply:
Yeah, I simply copied that from MySQL's official Docker docs, there wasn't any specific intent involved with it.